text/0032-nft-standard.md

```diff
@@ -93,13 +122,66 @@ The AttributeObject has the following JSON structure:
 
 The AttributeObject may have more attributes than the ones described above but those are the required ones.
 
+- The RoyaltyObject has the following JSON structure:
+
+```
+"creator_royalty_basis_points": {
+    "type": "integer",
+    "description": "Royalties percentage awarded to creators (basis points), e.g. 500"
+},
+"creators": {
+    "type": "Array<CreatorObject>",
+    "description": "Array of all creators involved in the project & their royalty bps."
+}
+```
+500 Basis Points mean 5% royalty, One basis point is equal to 1/100th of 1%. More info on BPS at [Investopedia](https://www.investopedia.com/terms/b/basispoint.asp)
+
+- The RoyaltyObject<**CreatorObject**> has the following JSON structure:
+
+```
+{
+    "address": {
+        "type": "string",
+        "description": "Address of the wallet in which royalty fees will be sent for secondary sales."
+    },
+    "share": {
+        "type": "integer",
+        "description": "Amount of royalty for each creator in bps, totaling a sum of 100."
+    }
+}
+```
+
+The CreatorObject may have more addresses than the ones described above, and the royalties will be split as per the share stated.
+
+- The InfoObject has the following JSON structure:
+
+```
+"animation_url": {
+    "type": "url",
+    "description": "URL to a non-immutable file hosted on CDN, given priority if the immutable file type is 3D object or isn't curently supported by the exchange or wallet preview."
+},
+"external_url": {
+    "type": "url",
+    "description": "URL to external website of the related project, e.g. Game website of the NFT asset."
+},
+"custom_fields": {
```

Comment:
Why is this relevant? Anyone can add anything to the JSON metadata, the standard is just to create a model so anyone integrating with the NFTs can know how to find some information. If we feel those custom fields are important for the marketplaces, then I feel we must have an object with them, not a free field without a clear description. 

In the example below you don't have the key `custom_fields` but you have

```
       "dna": "6c191e41-5bfe-432f-9e4b-ddcc718dce92",
       "discord": "https://discord.gg/87bYu52gDW"
```

Maybe they could be

```
"custom_information": {
       "dna": "6c191e41-5bfe-432f-9e4b-ddcc718dce92",
       "discord": "https://discord.gg/87bYu52gDW"
}
```

So it's easier for integrations.